salt/journal-cms/processes.sls

```diff
@@ -30,27 +30,16 @@ journal-cms-processes-start:
 {% set controller = "journal-cms-processes" %}
 
 {{ controller }}-script:
-    file.managed:
+    file.absent:
         - name: /opt/{{ controller }}.sh
-        - source: salt://elife/config/etc-init-multiple-processes-parallel.conf
```

Comment:
https://github.com/elifesciences/builder-base-formula/blob/master/elife/config/etc-init-multiple-processes-parallel.conf is clearly an Upstart script running Upstart commands, so it's right that it can't be used here on 16.04.